packages/app/pages/index.tsx

```diff
@@ -89,25 +273,27 @@ const Home: NextPage = () => {
         >
           <div style={{ minHeight: 64, padding: 20 }}>
             <Link href="/">
-              <a>Passports</a>
+              <a>NFT Passports Dashboard</a>
             </Link>
           </div>
+          <Tab to={"memberships"}>Memberships</Tab>
+          <Tab to={"users"}>Users</Tab>
+          <Tab to={"settings"}>Settings</Tab>
```

Comment:
we could abstract these tabs out to a constant array and loop once we have more clarity on what they should be and if design even wants to stick with a sidebar-tabbed dashboard anyway